src/components/form/Button.jsx

```
@@ -8,14 +8,19 @@ const Button = ({
   title,
   width = false,
   spinner = null,
+  bgColor
```

ESLint

[error] 'bgColor' is missing in props validation

(react/prop-types)